bgpd/bgp_zebra.c

```diff
@@ -662,14 +664,14 @@ zebra_read_ipv4 (int command, struct zclient *zclient, zebra_size_t length,
       if (bgp_debug_zebra((struct prefix *)&p))
 	{
 	  char buf[2][INET_ADDRSTRLEN];
-	  zlog_debug("Rx IPv4 route add VRF %u %s[%d] %s/%d nexthop %s metric %u tag %"ROUTE_TAG_PRI,
+	  zlog_debug("Rx IPv4 route add VRF %u %s[%d] %s/%d nexthop %s metric %u tag %d distance %u"ROUTE_TAG_PRI,
```

Comment:
This debug is broken with this change.  It should be
.... %u tag %" ROUTE_TAG_PRI " distance %u",